### ISSUE TITLE
Upgrade Ruby to Latest

### ISSUE THREAD

**Issue by murny** [ualbertalib/discovery#1432]
We talked about this for a few months now but nothing has been done yet about it. Here's an issue to track this.

We are using Ruby 2.1.5 in production. This version was released over 4 years ago! It is now unsupported and has been End of Life well over a year now! What does this mean? This means our Ruby version receives no more updates to it. This should be a big security concern since this version receives no security updates to it (which there has been quite a few present since this version has been EOL: https://www.ruby-lang.org/en/security/). There is also some major performance gains too from updating.

This is also prevented us from using the latest of other software like bootstrap and sass: https://github.com/ualbertalib/discovery/pull/1431#discussion_r241870723

What needs to be done? Nothing from the development side. Discovery works fine with the latest ruby version. We need to update the ansible playbook and bump this version.

(On a similar note we are using very old passenger and apache versions too which we tell every end user about this via our response headers (noted here as we do the same thing in jupiter: https://github.com/ualbertalib/jupiter/issues/703)

**Comment by pgwillia** [comment]
Latest is 2.6.  While current code base works well with 2.5 there appears to be a bit of work to get it going with [2.6](https://travis-ci.org/ualbertalib/discovery/builds/476933977).

**Comment by murny** [comment]
Agreed. When I created this issue 2.5 was the latest. So I would stick with `2.5.X`. 

Version 2.6  came out on Christmas so probably not quite ready to move to this version yet.
